core/spec/lib/spree/event_spec.rb

```diff
@@ -40,11 +40,12 @@ def toggle
           @run = true
         end
       end.new
-      subject.subscribe('foo', adapter: bus) { dummy.toggle }
+      subject.subscribe('foo', adapter: bus) { dummy.toggle; :done }
 
-      subject.fire 'foo', adapter: bus
+      firing = subject.fire 'foo', adapter: bus
 
       expect(dummy.run).to be(true)
+      expect(firing.executions.first.result).to be(:done)
```

Comment:
given we're now having multiple expectations, I think it makes sense to add the `:aggregate_failures` metadata here.

Reply:
Makes sense. Done.

Reply:
As you recommended in https://github.com/solidusio/solidus/pull/4204#discussion_r755859877, I've changed to use metadata instead of blocks.